coreference/bert_stuff_new.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
+# Author: Rehan
+import pickle
+#from preprocess import extract_mentions_aida
+import os.path
+import pyhocon
+import torch.nn as nn
+import torch
+import torch.nn.functional as F
+from models import *
+import torch
+from tqdm import tqdm
+config_file_path = '../cdlm/config_pairwise_long_reg_span.json'
+config = pyhocon.ConfigFactory.parse_file(config_file_path)
+#config['cdlm_path'] =  '../cdlm/models/cdlm2/'
+print(config.cdlm_path)
+
+
+
+def generate_cdlm_embeddings_from_model(parallel_model,bert_sentences, device, batch_size=150):
```

Comment:
clean this entire function. there are too many spaces... unnecessary commented code lines. unnecessary spaces.. always try to have no more than 1 line space between lines in a function. try adding documentation to the blocks of codes doing specific thing

Reply:
Yes, thanks for reviewing, will clean it up !